packages/ember/tests/dummy/app/templates/docs/coordination.md

```diff
@@ -0,0 +1,68 @@
+
+This means you can easily have more than one conceptual 'thing' happening at a time in your application code during testing, and you may want to work with milestones in more than one of those logical threads at a time.
+
+### Working with Multiple Coordinators
+
+As a general rule, you typically want one milestone coordinator for each logical thread of execution you're testing together. Put another way, **the milestones for each conceptual flow of control under test should be activated in a separate `activateMilestones` call**.
+
+```ts
+activateMilestones([MilestonesFor, ABackgroundPollingOperation]);
+activateMilestones([OtherMilestones, ForTimingGrowlStyle, UserNotifications]);
+```
+
+When you call the `advanceTo` function importable from `@milestones/core`, it will automatically delegate to the active coordinator for the key you give it.
+
+Alternatively, each `activateMilestones` call returns the `MilestoneCoordinator` instancec it creates. If you're dealing with very fine-grained interactions between the flows you're testing, though, and moving between them often, you may find your test code easier to follow if you invoke the `advanceTo` method on the corresponding coordinator itself.
+
+```ts
+let pollThread = activateMilestones([/*...*/]);
+let notificationThread = activateMilestones([/*...*/]);
+
+// ...
+
+await pollThread.advanceTo('fetch').andReturn({ dataChanged: true });
+await notificationThread.advanceTo('will-show-change-notification').andContinue();
+
+// assert that the notification is visible
+```
+
+## Designating Default Behavior
+
+When activating a set of milestones, it's also possible to designate a default behavior for those milestones.
+
+```ts
+activateMilestones([MyMilestone], {
+  onMilestoneReached(milestone) {
+    milestone.return();
+  }
+});
+```
+
+By specifying an `onMilestoneReached` callback for a coordinator, it can automatically handle any milestones under its jurisdiction rather than requiring that they be explicitly advanced to and then dealt with as part of your test logic. This allows you to effectively neutralize bits of behavior in your application code that aren't under test and might otherwise unnecessarily slow things down or otherwise make them unpredictable.
+
+Note that if you _do_ explicitly `advanceTo` a milestone that belongs to a coordinator with a default handler, that handler won't be applied and you'll instead be able to decide what it does yourself, as long as you call `advanceTo` before that milestone is reached.
```

Comment:
> as long as you call `advanceTo` before that milestone is reached

This is confusing in that milestones is a tool to manipulate timing, but this sounds like you have to call `advanceTo` "quickly".  Are there any tips to ensure you call `advanceTo` at the right time for a milestone with a default handler, or will this mostly just work out timing-wise?

Reply:
The issue with calling `advanceTo` on a milestone that has a default handler is that you're essentially racing with yourself—if the milestone is reached before you call advance to, it will immediately invoke the default handler, because it has no way of knowing that you actually want to do something different.

I think maybe I should just leave this sentence out—it's probably better to do something like this anyway (which is a pattern described at the very bottom of this page):

```ts
activateMilestones(['some-shared-tag'], { onMilestoneReached: () ... });
activateMilestones([SpecificID]);
```